legend-sdlc-generation-file/src/main/java/org/finos/legend/sdlc/generation/artifact/ArtifactGenerationFactory.java

```diff
@@ -67,21 +74,44 @@ public MutableMap<ArtifactGenerationExtension, List<ArtifactGenerationResult>> g
         MutableMap<ArtifactGenerationExtension, List<ArtifactGenerationResult>> results = Maps.mutable.empty();
         for (PackageableElement element : this.elements)
         {
-            org.finos.legend.pure.m3.coreinstance.meta.pure.metamodel.PackageableElement packageableElement = pureModel.getPackageableElement(element.getPath());
-            for (ArtifactGenerationExtension extension : this.extensions)
+            org.finos.legend.pure.m3.coreinstance.meta.pure.metamodel.PackageableElement packageableElement = this.findPackageableElement(pureModel, element);
+            if (packageableElement != null)
             {
-                if (extension.canGenerate(packageableElement))
+                for (ArtifactGenerationExtension extension : this.extensions)
                 {
-                    List<Artifact> artifacts = this.generateArtifacts(packageableElement, element, extension);
-                    List<GenerationOutput> outputs = ListIterate.collect(artifacts, artifact -> new GenerationOutput(artifact.content, artifact.path, artifact.format));
-                    ArtifactGenerationResult result = new ArtifactGenerationResult(packageableElement, outputs, extension);
-                    results.getIfAbsentPut(result.getGenerator(), Lists.mutable::empty).add(result);
+                    if (extension.canGenerate(packageableElement))
+                    {
+                        List<Artifact> artifacts = this.generateArtifacts(packageableElement, element, extension);
+                        List<GenerationOutput> outputs = ListIterate.collect(artifacts, artifact -> new GenerationOutput(artifact.content, artifact.path, artifact.format));
+                        ArtifactGenerationResult result = new ArtifactGenerationResult(packageableElement, outputs, extension);
+                        results.getIfAbsentPut(result.getGenerator(), Lists.mutable::empty).add(result);
+                    }
                 }
             }
         }
         return results;
     }
 
+    private org.finos.legend.pure.m3.coreinstance.meta.pure.metamodel.PackageableElement findPackageableElement(PureModel pureModel, PackageableElement packageableElement)
+    {
+        org.finos.legend.pure.m3.coreinstance.meta.pure.metamodel.PackageableElement element = pureModel.getPackageableElement_safe(packageableElement.getPath());
+        if (element != null)
+        {
+            return element;
+        }
+        if (packageableElement instanceof Function)
+        {
+            Function elementFunc = (Function) packageableElement;
+            String fullPath = pureModel.buildPackageString(packageableElement._package, HelperModelBuilder.getSignature(elementFunc));
+            element = pureModel.getPackageableElement_safe(fullPath);
+        }
+        if (element == null)
+        {
+            LOGGER.info("Unable to find element '" + packageableElement.getPath() + "' in Pure Model");
```

Comment:
I think we ought to log this at debug level.
```suggestion
            LOGGER.debug("Unable to find element '{}' in Pure Model", packageableElement.getPath());
```